Scrape.py

```diff
@@ -85,14 +127,11 @@ def pull_image_urls(url, board):
 		print(e.args)
 		return
 
-	noPosts = len(json_data)
-
-	for x in range(0, noPosts):
-		posts = json_data[x]
-		if posts.get("tim") is not None:
-			currentImgURL = str(posts.get("tim")) + posts.get("ext")
-			img_URLS.append(currentImgURL)
-	print(str(len(img_URLS)) + " Images Found")
+	JSON=JsonWorker(json_data)
+	global img_URLS
+	img_URLS=JSON.getImageURLs()
+	global path
```

Comment:
Probably worth refactoring so that these don't have to be global

Reply:
It's been so long I don't even what I changed. I'll take a look at this at some point.

Reply:
No worries, its an old PR 😆